api/vpc/v1alpha2/vpcpeering_types.go

```diff
@@ -151,6 +151,9 @@ func (peering *VPCPeering) Validate(ctx context.Context, kube client.Reader, fab
 		return nil, errors.Wrapf(err, "failed to validate metadata")
 	}
 
+	if fabricCfg == nil {
```

Comment:
Validation shouldn't fail if no fabricCfg is supplied  - it just skips dependent checks, so, It should be smth like `if fabricCfg != nil && fabricCfg.VPCPeeringDisabled`